vector-im/element-android
# Adds NewAppLayoutEnabled Feature Flag

## Type of change


## Content

Adds `new_app_layout_enabled` Feature Flag

## Motivation and context

Closes https://github.com/vector-im/element-android/issues/6578

We will use this feature flag to hide changes related to the Space Switching and Edit Layout projects until they are both ready to be released together

The flag can be found in debug menu > private settings

## Screenshots / GIFs

<img src="https://user-images.githubusercontent.com/20701752/179714082-93cfc789-b93a-43eb-9b5e-f23fa3696577.png" width="200" />


## Tests

- Go to the debug menu (either inside hamburger menu or first screen on auth flow)
- Go to features
- Enable feature flag
- Restart app and see feature flag is still enabled

## Tested devices

- OS version(s): Android 12

## Checklist
